docs/tutorial/sandbox.md

```diff
@@ -12,28 +12,8 @@ the GPU service and the network service.
 
 See Chromium's [Sandbox design document][sandbox] for more information.
 
-## Electron's sandboxing policies
-
-Electron comes with a mixed sandbox environment, meaning sandboxed processes can run
-alongside privileged ones. By default, renderer processes are not sandboxed, but
-utility processes are. Note that as in Chromium, the main (browser) process is
-privileged and cannot be sandboxed.
-
-Historically, this mixed sandbox approach was established because having Node.js available
-in the renderer is an extremely powerful tool for app developers. Unfortunately, this
-feature is also an equally massive security vulnerability.
-
-Theoretically, unsandboxed renderers are not a problem for desktop applications that
-only display trusted code, but they make Electron less secure than Chromium for
-displaying untrusted web content. However, even purportedly trusted code may be
-dangerous — there are countless attack vectors that malicious actors can use, from
-cross-site scripting to content injection to man-in-the-middle attacks on remotely loaded
-websites, just to name a few. For this reason, we recommend enabling renderer sandboxing
-for the vast majority of cases under an abundance of caution.
-
-<!--TODO: update this guide when #28466 is either solved or closed -->
-Note that there is an active discussion in the issue tracker to enable renderer sandboxing
-by default. See [#28466][issue-28466]) for details.
+**As of Electron 20, all renderer and utility processes in Electron are sandboxed by default.**
```

Comment:
This isn't quite true—specifying `nodeIntegration: true` still causes the process to be non-sandboxed. I guess you could argue that this is still "sandboxed by default", but it doesn't involve the `sandbox` option, so I think it probably deserves clarification. Also to avoid giving the idea that processes with node integration are / could be "sandboxed", since "processes are sandboxed by default".

Reply:
[8411bdd](https://github.com/electron/electron/pull/35379/commits/8411bdd40f057c8cfc16d10af326c5969c90615e)